onnx/hub.py

```diff
@@ -241,10 +240,7 @@ def load(
 
     if force_reload or not os.path.exists(local_model_path):
         if not _verify_repo_ref(repo) and not silent:
-            msg = (
-                'The model repo specification "{}" is not trusted and may'
-                " contain security vulnerabilities. Only continue if you trust this repo."
-            ).format(repo)
+            msg = f'The model repo specification "{repo}" is not trusted and may contain security vulnerabilities. Only continue if you trust this repo.'
```

Comment:
```suggestion
            msg = f"The model repo specification "{repo}" is not trusted and may contain security vulnerabilities. Only continue if you trust this repo."
```
Double quote is preferred.

Reply:
Understood.